playbooks/set-ceph-config.yml

```diff
@@ -0,0 +1,26 @@
+- name: Set ceph config "{{ option }}" to "{{ value }}"
```

Comment:
Variables won't work in name. use `"option"` & `"value"`

Reply:
Done, made the change